src/Algebra/NLSolvers.jl

```diff
@@ -52,7 +52,7 @@ function _solve_nr!(x,A,b,dx,ns,nls,op)
 
   # Newton-like iterations
   for nliter in 1:nls.max_nliters
-
```

Comment:
Remove this (better start from scratch as indicated above)

Reply:
removed in clean version